From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@electronjs.org>
Date: Thu, 26 May 2022 10:45:49 -0400
Subject: allow disabling of v8 sandboxed pointers

V8 sandboxed pointers are incompatible with Node due to changes like
https://chromium-review.googlesource.com/c/v8/v8/+/3114136, so for
now we need a way to disable v8 sandboxed pointers.

diff --git a/BUILD.gn b/BUILD.gn
index 68249e1299a483df48704862c35ce211a4a70e91..f39e93fdbae347e22864338df996d2d04eee0aee 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -506,9 +506,9 @@ if (v8_enable_sandbox == "") {
       v8_enable_external_code_space && target_os != "fuchsia"
 }
 
-# Enable sandboxed pointers on desktop when the sandbox is enabled.
+# Disable sandboxed pointers.
 if (v8_enable_sandbox) {
-  v8_enable_sandboxed_pointers = target_os != "android"
+  v8_enable_sandboxed_pointers = false
 }
 
 # Enable all available sandbox features if sandbox future is enabled.
